src/entities/createPool/utils/validateInputs.ts

```diff
@@ -0,0 +1,35 @@
+import { CreatePoolInput } from '../types';
+
+export class ValidateInputs {
```

Comment:
Any reason to create a ValidateInputs class? We've been using a function on add/remove liquidity and I think it would be nice to keep the same standard between them. What do you think?